docs/features/workspaces.md

```diff
@@ -25,53 +25,50 @@ By default, all workspaces are public. This means that anyone with the link to y
 
 The expandable sidebar includes:
 
- - Options for creating and viewing workspaces.
- - Access to your Contributor and Repository Insights pages.
- - Links to support.
+- Options for creating and viewing workspaces.
+- Access to your Contributor and Repository Insights pages.
+- Links to support.
 
- ![Sidebar](../../static/img/workspace-sidebar.png)
+![Sidebar](../../static/img/workspace-sidebar.png)
 
-:::tip
+:::info
 
- We recently had a naming change. The Contributors Insights section was previously called [Lists](contributor-insights.md), and the Repositories Insights section was previously called [Insights](./repo-insights.md).
+We recently had a naming change. The [Contributors Insights](contributor-insights.md) section was previously called Lists, and the [Repositories Insights](./repo-insights.md) section was previously called Insights.
 
 :::
 
-
 ### Your Workspaces
 
-In your workspace dashboards, you can create, view, and manage your workspaces. 
+In your workspace dashboards, you can create, view, and manage your workspaces.
 
 #### Repositories Dashboard
 
 ![Workspaces](../../static/img/workspace.png)
 
 Within the repositories dashboard, you can view the following metrics, which are over a period of thirty days by default:
 
-- **Pull Requests**: This includes the total Opened and Merged Pull Requests for the repositories included in your workspace, as well as the velocity of pull requestions being merged.
-- **Issues**: This includes the total Opened and Closed Issues for the repositories included in your workspace, as well as the velocity of issues being closed.
+- **Pull Requests**: This includes the total opened and merged pull requests for the repositories included in your workspace, as well as the velocity of pull requests being merged.
+- **Issues**: This includes the total opened and closed issues for the repositories included in your workspace, as well as the velocity of issues being closed.
 - **Engagement**: This includes the total stars, forks, and activity ratio for the repositories included in your workspace.
 
-
 ## How to Create or Access a Workspace
 
 To create a workspace, start by accessing the **Workspace switcher** in the sidebar. From there, you can create a new workspace or view an existing one.
 
-:::tip
+:::note
 
-Whenever you select Home, you'll be taken to the last workspace you opened.
+Whenever you select "Home", you'll be taken to the last workspace you opened.
 
 :::
 
 ### How to Add Repositories to a Workspace
 
 If you're creating a new workspace, you'll have the option to add repositories at the time of creation.
 
-Start by creating a Workspace name, and then click `Add repositories`. You'll have three options for adding repositories:
+Start by creating a workspace name, then click "Add repositories." You'll have three options for adding repositories:
```

Comment:
I think there are two issues here: Maybe the issue here is "How to Add repositories to a workspace" and "How to Add Repos to Your Personal Workspace."

We can have two separate lines.

- **Adding repos to a workspace**- {description}
- **Adding repos to your personal workspace** - {Deseciption}

I _believe_ that only the personal workspace has the edit button. That's because it also takes you to the settings page where you can add workspace members, change the visibility, and upgrade. Which is something I'm going to add to the docs after we get this merged in.

Reply:
Ah, now I see. You're right. Only "Your personal workspace" that has the edit button in the beginning.
Creating new workspace will take us the creation form as below.

![Screenshot 2024-03-04 204657](https://github.com/open-sauced/docs/assets/45172775/28af21e2-ca34-42e6-9bf8-f208cef77e62)

I remember that you will add more details to the workspaces, but I'm not sure if this one of them. So, I throw this here, just in case. 😅

Reply:
I think this could be a separate issue. What do you think @BekahHW?